Sources/Public/Animation/LottieView.swift

```diff
@@ -287,6 +304,18 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a new instance of this view, which will invoke the provided `loadAnimation` closure
+  /// whenever the `binding` value is updated.
+  ///
+  /// - Note: This function requires a valid `loadAnimation` closure provided during view initialization.
```

Comment:
```suggestion
  /// - Note: This function requires a valid `loadAnimation` closure provided during view initialization,
  ///         otherwise the `loadAnimationTrigger` will have no effect.
```